1-mandatory/1-writers.js

```diff
@@ -62,7 +66,14 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  let deadWritersIn40s = writers.filter((writer) => {
```

Comment:
Nitpick: It would be slighly cleaner to chain the functions. e.g.
```
writers
  .filter(writer => writer.age > 39 && writer.age < 50 && !writer.alive)
  .map(writer => `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`)
  .forEach(writerString => console.log(writerString ));
```

Reply:
now chained - yes code requires less temporary variables this way. Thanks